docker_test.go

```diff
@@ -181,3 +211,25 @@ func TestDockerVersionScript(t *testing.T) {
 		t.Errorf("Could not get docker version, error: %s", err.Error())
 	}
 }
+
+func TestDockerFatManifestPull(t *testing.T) {
```

Comment:
How is this test different from the existing TestDockerPull? Why is it needed?

Reply:
Docker pull works with a non-fat-manifest image, unlike TestDockerFatManifestPull.
The idea behind this test is to validate the build info while pulling a fat-manifest-image from the remote repo.
The problem with remote repo is that Artifactory makes a different folder structure for pulling fat-manifest-images  witch break the collect build info step